### DISCUSSION TITLE
SVUnit Merge

### DISCUSSION THREAD

**Discussion by eminakgun** [svunit/svmock#23] [General]
Is this repo gonna make its way through the original SVUnit? As far as I see, there are no implemented mocking mechanisms in https://github.com/svunit/svunit other than uvm report mock.

**Comment by tudortimi** [comment]
I'm not sure whether it's a good idea to merge it into SVUnit, because then the library becomes bloated. It would be a good idea though to move it to the svunit organization at least.

**Reply by eminakgun** [reply]
I thought SVUnit is a suite of unit testing utilities, and mocking is an essential one.
